gr-utils/python/modtool/modtool_add.py

```diff
@@ -110,7 +112,13 @@ def setup(self, options, args):
         if not options.license_file:
             self._info['copyrightholder'] = options.copyright
             if self._info['copyrightholder'] is None:
-                self._info['copyrightholder'] = '<+YOU OR YOUR COMPANY+>'
+                user = getpass.getuser()
+                git_user = subprocess.check_output('git config --global user.name', shell=True).strip()
```

Comment:
This assumes the existence of git. It's a fair assumption for GNU Radio devs, but not good practice for code in general.

Reply:
...and maybe it's a better idea to put this log into `scm.py`